modules/graph_ql/main.tf

```diff
@@ -0,0 +1,71 @@
+
+data "aws_iam_policy_document" "graphql_task_policy_document" {
+  statement {
+    actions   = ["sts:AssumeRole"]
+    resources = ["*"]
```

Comment:
This is fairly open with `resources = ["*"]`. I think for initial testing this is OK but we should plan on either now or via a card to come back and specify the resources that are actually needed for a least privileged approach.

Reply:
I am presently unable to find a good example of how to restrict an AssumeRole policy. Created https://bugs.earthdata.nasa.gov/browse/ORCA-576